<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidget</class>
 <widget class="QWidget" name="LogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="toolbar">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ðŸ“‹ æ—¥å¿—</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="levelLabel">
       <property name="text">
        <string>çº§åˆ«:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="levelCombo">
       <item>
        <property name="text">
         <string>DEBUG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>INFO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WARNING</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ERROR</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoScrollCheck">
       <property name="text">
        <string>è‡ªåŠ¨æ»šåŠ¨</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>æ¸…é™¤</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>å¯¼å‡º</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logText">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
    background-color: #f8f8f8;
    border: 1px solid #ddd;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
